zetta_utils/builder/build.py

```diff
@@ -19,6 +20,19 @@
     "version": "@version",
 }
 
+BUILT_OBJECT_ID_REGISTRY: dict[int, JsonDict] = {}
+
+
+def get_initial_builder_spec(obj: Any) -> JsonDict | None:
+    """Returns the builder spec that the object was initially built with.
+    Note that mutations to the object after it was built will not be
+    reflected in the spec. Returns `None` if the object was not built with
+    builder
+    """
+    # breakpoint()
+    result = BUILT_OBJECT_ID_REGISTRY.get(id(obj), None)
```

Comment:
Could remove unnecessary result variable and breakpoint comment; looks good otherwise.